luarules/gadgets/unit_inherit_creation_xp.lua

```diff
@@ -59,7 +58,6 @@ local function calculatePowerDiffXP(childID, parentID) -- this function calculat
 end
 
 local initializeList = {}
```

Comment:
Nothing adds to this list

Reply:
It should be added to in the UnitFinished callin further down.